velodrome-finance/automations
# Gauge upkeep manager

Part of #1 

Change log:
- Setup hardhat project
- Add `CronUpkeepManager` contract to create and fund new cron upkeeps
- Add deploy script and temporary steps for live testing

Manual test:
1. Setup env variables. Example values are provided for OP Sepolia.
2. Fund the deployer address with at least 2 testnet LINK. Use this faucet: https://faucets.chain.link
3. Run the deploy script `npx hardhat run scripts/deploy.ts --network optimismSepolia`
4. The script will output the automation URL for the UI management console
5. Check if the automation is triggered every 15 mins
